### PR TITLE
Add note guidelines for additional distributions

### PR DESCRIPTION
**Pull request by slaukilo**
##### SUMMARY
Addressing issue [80353](https://github.com/ansible/ansible/issues/80353) by adding two notes on `installation_distros.rst` regarding outside project installation instructions

##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### ISSUE TYPE

- Docs Pull Request

### CODE REVIEW

File: docs/docsite/rst/installation_guide/installation_distros.rst [hunk 1]
```diff
@@ -2,11 +2,14 @@
 
 Installing Ansible on specific operating systems
 ================================================
+.. note:: These instructions are provided by their respective communities. Any bugs/issues should be filed with that community to update these instructions. Ansible maintains only the pip install instructions.
```
Comment (webknjaz) [inline docs/docsite/rst/installation_guide/installation_distros.rst]: I'm pretty sure there must be an empty line after the title.
```suggestion

.. note:: These instructions are provided by their respective communities. Any bugs/issues should be filed with that community to update these instructions. Ansible maintains only the ``pip install`` instructions.
```

File: docs/docsite/rst/installation_guide/installation_distros.rst [hunk 2]
```diff
@@ -2,11 +2,14 @@
...
 
 The ``ansible`` package can always be :ref:`installed from PyPI using pip <intro_installation_guide>` on most systems but it is also packaged and maintained by the community for a variety of Linux distributions.
 
 The following instructions will guide you through installing the ``ansible`` package with your preferred distribution's package manager.
 
+.. note:: For maintainers wanting to add distributions not mentioned within the guide, install instructions are included here only for distributions with reasonably up to date version of ansible and follow Ansible policies and version constraints. The distribution MUST ensure that ansible-core and ansible are synced.Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the Ansible Packaging <https://matrix.to/#/#packaging:ansible.com_ matrix room.
```
Comment (webknjaz) [inline docs/docsite/rst/installation_guide/installation_distros.rst]: This seems to have a `>` and backticks missing.
```suggestion
.. note:: For maintainers wanting to add distributions not mentioned within the guide, install instructions are included here only for distributions with reasonably up to date version of ``ansible`` and follow Ansible policies and version constraints. The distribution MUST ensure that ``ansible-core`` and ``ansible`` are synced. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ matrix room.
```

File: docs/docsite/rst/installation_guide/installation_distros.rst (hunk 1 above)
Comment (slaukilo) [reply docs/docsite/rst/installation_guide/installation_distros.rst]: I believe you are correct, now looking at the reStructured syntax in comparison to markdown.

File: docs/docsite/rst/installation_guide/installation_distros.rst (hunk 2 above)
Comment (slaukilo) [reply docs/docsite/rst/installation_guide/installation_distros.rst]: >This seems to have a `>` and backticks missing.

Will incorporate syntax corrections and be cognizant of markup syntax going forward.

**Comment by gotmax23** [comment]
Thanks for submitting this, @slaukilo! I'll review this later.

File: docs/docsite/rst/installation_guide/installation_distros.rst [hunk 3]
```diff
@@ -3,10 +3,14 @@
 Installing Ansible on specific operating systems
 ================================================
 
+.. note:: These instructions are provided by their respective communities. Any bugs/issues should be filed with that community to update these instructions. Ansible maintains only the ``pip install`` instructions.
+
 The ``ansible`` package can always be :ref:`installed from PyPI using pip <intro_installation_guide>` on most systems but it is also packaged and maintained by the community for a variety of Linux distributions.
 
 The following instructions will guide you through installing the ``ansible`` package with your preferred distribution's package manager.
 
+.. note:: For maintainers wanting to add distributions not mentioned within the guide, install instructions are included here only for distributions with reasonably up to date version of ``ansible`` and follow Ansible policies and version constraints. The distribution MUST ensure that ``ansible-core`` and ``ansible`` are synced. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ matrix room.
```
Comment (oraNod) [inline docs/docsite/rst/installation_guide/installation_distros.rst]:
```suggestion
.. note:: For maintainers wanting to add distributions not mentioned within the guide, install instructions are included here only for distributions with reasonably up to date version of ``ansible`` and follow Ansible policies and version constraints. The distribution MUST ensure that ``ansible-core`` and ``ansible`` versions are kept in sync. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ matrix room.
```

Comment (oraNod) [inline docs/docsite/rst/installation_guide/installation_distros.rst]:
```suggestion
.. note:: For maintainers wanting to add distributions not mentioned within the guide, install instructions are included here only for distributions with reasonably up-to-date version of ``ansible`` and follow Ansible policies and version constraints. The distribution MUST ensure that ``ansible-core`` and ``ansible`` are synced. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ matrix room.
```

**Comment by gotmax23** [comment]
> and follow Ansible policies and version constraints

I'd prefer to remove this part even though I think I originally suggested it :). We don't currently have any policies that would apply to distribution packages and "keep in sync" covers the version constraint part.

**Comment by gotmax23** [comment]
``` rst
For maintainers wanting to add distributions not mentioned within the guide
```

would be more concise as

``` rst
For maintainers who wish to add distributions to this guide
```

or something similar

File: docs/docsite/rst/installation_guide/installation_distros.rst (hunk 3 above)
Comment (slaukilo) [reply docs/docsite/rst/installation_guide/installation_distros.rst]: wording selection really does make the difference, I'll keep it concise in the future.

**Comment by slaukilo** [comment]
Going sentence-by-sentence its apparent how clunky my wording was, I agree with both revisions and will incorporate the changes. Thanks for your patience.

File: docs/docsite/rst/installation_guide/installation_distros.rst [hunk 4]
```diff
@@ -3,10 +3,14 @@
 Installing Ansible on specific operating systems
 ================================================
 
+.. note:: These instructions are provided by their respective communities. Any bugs/issues should be filed with that community to update these instructions. Ansible maintains only the ``pip install`` instructions.
+
 The ``ansible`` package can always be :ref:`installed from PyPI using pip <intro_installation_guide>` on most systems but it is also packaged and maintained by the community for a variety of Linux distributions.
 
 The following instructions will guide you through installing the ``ansible`` package with your preferred distribution's package manager.
 
+.. note:: For maintainers who wish to add distributions to this guide, installation instructions are included here only for distributions with a reasonably up-to-date version of ``ansible``. The distribution MUST ensure that ``ansible-core`` and ``ansible`` versions are kept in sync to the extent that their build system allows. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ matrix room.
```
Comment (gotmax23) [inline docs/docsite/rst/installation_guide/installation_distros.rst]:
```suggestion
.. note:: For maintainers who wish to add distributions to this guide, installation instructions are included here only for distributions with a reasonably up-to-date version of ``ansible``. The distribution MUST ensure that ``ansible-core`` and ``ansible`` versions are kept in sync to the extent that the distribution build system allows. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ matrix room.
```

**Comment by gotmax23** [comment]
> Going sentence-by-sentence its apparent how clunky my wording was, I agree with both revisions and will incorporate the changes. Thanks for your patience.

It's no problem :). Thank you for responding to the feedback.

File: docs/docsite/rst/installation_guide/installation_distros.rst (hunk 4 above)
Comment (slaukilo) [reply docs/docsite/rst/installation_guide/installation_distros.rst]: I need to squash the vagueness is my writing, direct reference always sounds better. Thanks :)

Comment (gotmax23) [reply docs/docsite/rst/installation_guide/installation_distros.rst]: It was [my suggestion](https://github.com/ansible/ansible/pull/80389/#issuecomment-1499239076) to use `they`, but then I realized that pronoun was confusing :).

**Comment by gotmax23** [comment]
> > @slaukilo This PR contains `@` mentions in at least one commit message. Those mentions can cause cascading notifications through GitHub and need to be removed. Please squash or amend your commits to remove the mentions.
> 
> You can replace the current commit body with
> 
> ```
> Co-authored-by: Maxwell G <maxwell@gtmx.me>
> ```

`Co-authored-by: Maxwell G <maxwell@gtmx.me>`should be on its own line. It's a trailer.

**Comment by slaukilo** [comment]
Amended commit also merged upstream causing failure, will rebase and apply changes.

**Comment by slaukilo** [comment]
As current branch is up-to-date with `devel` (can't rebase), will a `git revert` to a reflog HEAD be most appropriate at this time?

**Comment by gotmax23** [comment]
I fixed it for you in my fork[^1]. You can run these commands to reset your branch:

```
git remote add gotmax23 https://github.com/gotmax23/ansible
git fetch gotmax23
git reset --hard gotmax23/80353-update-installation-srt
git push --force-with-lease
```

[^1]: I checked out your PR, ran `git reset --hard 5c9d8ad07b9a2883c27d40f36785cd244bc5e281`, did a `git rebase -i` to correct the `Revise/apply grammar change` commit body, rebased on top of `devel`, and pushed to my fork. https://git-rebase.io/ is a good tutorial if you're interested.

**Comment by slaukilo** [comment]
> I checked out your PR, ran git reset --hard 5c9d8ad07b9a2883c27d40f36785cd244bc5e281, did a git rebase -i to correct the Revise/apply grammar change commit body, rebased on top of devel, and pushed to my fork. https://git-rebase.io/ is a good tutorial if you're interested. 

I made the mistake of rebasing to the upstream branch instead of the individual hash and then executing a `git reset` to the incorrect `git reflog` HEAD which only worsened with each commit revert. Reading through the link you shared helped me to recognize the error in my approach. I'll read up more on git before I try to tackle non-doc issues. Thank you for your continued guidance and support in this issue!

**Review by samccann — APPROVED** [review]

File: docs/docsite/rst/installation_guide/installation_distros.rst [hunk 5]
```diff
@@ -3,10 +3,14 @@
...
 ================================================
 
+.. note:: These instructions are provided by their respective communities. Any bugs/issues should be filed with that community to update these instructions. Ansible maintains only the ``pip install`` instructions.
+
 The ``ansible`` package can always be :ref:`installed from PyPI using pip <intro_installation_guide>` on most systems but it is also packaged and maintained by the community for a variety of Linux distributions.
 
 The following instructions will guide you through installing the ``ansible`` package with your preferred distribution's package manager.
 
+.. note:: For maintainers who wish to add distributions to this guide, installation instructions are included here only for distributions with a reasonably up-to-date version of ``ansible``. The distribution MUST ensure that ``ansible-core`` and ``ansible`` versions are kept in sync to the extent that the distribution build system allows. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ matrix room.
```
Comment (gotmax23) [inline docs/docsite/rst/installation_guide/installation_distros.rst]:
```suggestion
.. note:: For maintainers who wish to add distributions to this guide, installation instructions are included here only for distributions with a reasonably up-to-date version of ``ansible``. The distribution MUST ensure that ``ansible-core`` and ``ansible`` versions are kept in sync to the extent that the distribution build system allows. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ Matrix room.
```

**Review by gotmax23 — APPROVED** [review]
Besides this one last nit, this looks good to me as well. Congratulations on your first contribution to ansible!

**Comment by samccann** [comment]
Thanks @slaukilo for the Ansible docs fix and welcome to the Ansible project!

**Pull request merged**